spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkWriteUtil.java

```diff
@@ -130,15 +130,16 @@ private static Distribution copyOnWriteDeleteUpdateDistribution(
 
   /** Builds requirements for merge-on-read DELETE, UPDATE, MERGE operations. */
   public static SparkWriteRequirements positionDeltaRequirements(
```

Comment:
There is a note at the top that this class is for internal purposes and has no API guarantees.